File: Python/blink.py
import RPi.GPIO as GPIO
import time

GPIO.setmode(GPIO.BOARD)

led = 11
GPIO.setup(led, GPIO.OUT)

try:
	while True:
		GPIO.output(led, 0)
		time.sleep(1)
		GPIO.output(led, 1)
		time.sleep(1)		
except KeyboardInterrupt:
	GPIO.cleanup()

File: Ruby/blink.rb
require 'wiringpi'

# initialize the GPIO port:
gpio = WiringPi::GPIO.new
# name the pins, for easier reference:
led = 0

# initialize the pin functions:
gpio.mode led, OUTPUT

# turn off the led, just in case:
gpio.write led, 0

begin
	loop do
		gpio.write led, 1
		sleep 1
		gpio.write led, 0
		sleep 1
	end
rescue SystemExit, Interrupt
	puts 'Bye'
end
